pkg/mobile/types_test.go

```diff
@@ -1,110 +1,103 @@
-package mobile
+package mobile_test
```

Comment:
@darahayes @aliok I've tried reusing the fixtures in the new `test` package from #51 here but it created a cyclical dependency, so I had to:

- create this separate `mobile_test` package
- export the static error messages from `mobile/types`

Personally I think it's a nice tradeoff and there's some [nice articles](https://medium.com/@benbjohnson/structuring-tests-in-go-46ddee7a25c) mentioning this approach, but it's different from what we have so far.

WDYT? Any other suggestions?

Reply:
I don't have a strong opinion on this. It looks good to me.